src/richie/apps/courses/cms_wizards.py

```diff
@@ -284,6 +284,7 @@ class CourseRunWizardForm(BaseWizardForm):
 
     model = CourseRun
 
+    # pylint: disable=invalid-overridden-method
```

Comment:
This one can be solved by using `@cached_property` instead of `@property` like it's done in the parent class.
The property is just cached in the instance so this is fine: https://github.com/django/django/blob/master/django/utils/functional.py#L7